### PR TITLE
Fix KeepMinDuplicatePokemon

### PR DESCRIPTION
**Pull request by skupfer**

### CODE REVIEW

**Comment by skupfer** [comment]
hopefully ... I tried it and it worked^^

**Comment by OverlordQ** [comment]
Since you're already passing _client, you dont need to create an entire new object and pass it as well, in `Inventory.cs` you could reference `_client.Settings` or in the `Inventory(Client)` constructor initialize `_clientSettings` from `client`

File: PokemonGo.RocketAPI.Logic/Inventory.cs [hunk 1]
```diff
@@ -68,7 +70,7 @@ public Inventory(Client client)
                         p.Where(x => x.Favorite == 0)
                             .OrderByDescending(x => x.Cp)
                             .ThenBy(n => n.StaminaMax)
-                            .Skip(1)
+                            .Skip(_clientSettings.KeepMinDuplicatePokemon)
```
Comment (marksterxxx) [inline PokemonGo.RocketAPI.Logic/Inventory.cs]: .Skip(_client.Settings.KeepMinDuplicatePokemon)

File: PokemonGo.RocketAPI.Logic/Inventory.cs [hunk 2]
```diff
@@ -13,9 +13,11 @@ namespace PokemonGo.RocketAPI.Logic
     public class Inventory
     {
         private readonly Client _client;
+        private readonly ISettings _clientSettings;
 
-        public Inventory(Client client)
+        public Inventory(ISettings clientSettings, Client client)
```
Comment (marksterxxx) [inline PokemonGo.RocketAPI.Logic/Inventory.cs]: no ne
no need

**Comment by marksterxxx** [comment]
@OverlordQ  Yes no need to pass _clientSettings
.Skip(_client.Settings.KeepMinDuplicatePokemon)

**Pull request merged**